activities/api.py

```diff
@@ -93,3 +95,26 @@ def perform_create(self, serializer):
             staff_member_id=serializer.validated_data['staff_member'].id,
             link=f'/panel/operations/evaluate/{operation.id}'
         )
+
+
+class ExamViewSet(ModelViewSet):
+    queryset = Exam.objects.all()
+    filter_backends = [DjangoFilterBackend]
+
+    filterset_fields = ['competence_level']
+
+    def get_serializer_class(self):
+        if self.action == 'list':
```

Comment:
please check if `self.request.mthod == "GET"` instead of  `self.action == 'list'`